Cargo.toml

```diff
@@ -18,6 +18,7 @@ regex = "1.4.2"
 serde_json = "1.0.59"
 teloxide = "0.3.3"
 tokio = { version = "0.2.22", features = ["rt-threaded", "macros"] }
+openssl = { version = "0.10", features = ["vendored"] }
```

Comment:
Why do we need this here? Another problem is that `vendored` version will not detect systems's root certificates. Also possibly some changes to the build process are required since openssl build requires C compiler, perl and other stuff (need to check it)